kubetest/bash.go

```diff
@@ -30,17 +31,22 @@ type bashDeployer struct {
 	gcpZone                 string
 	gcpSSHProxyInstanceName string
 	provider                string
+	nodeLoggingEnabled      bool
 }
 
 var _ deployer = &bashDeployer{}
 
+var (
+	bashNodeLogging = flag.Bool("bash-node-logging", false, "(bash only) enable node logging to gcp")
```

Comment:
what's one more global + global flag, among friends? :upside_down_face: 

@amwat, we should just keep this off in kubetest2, w/o the flag.